google-home-community.groovy

```diff
@@ -2259,6 +2300,7 @@ private attributesForTrait_Volume(deviceTrait) {
     return [
         volumeMaxLevel:         100,
         volumeCanMuteAndUnmute: deviceTrait.canMuteUnmute,
+        volumeCanSetVolume:     deviceTrait.canSetVolume,
```

Comment:
It doesn't look like Google has this attribute documented.  Their documentation says the `volumeRelative` command is only called if `commandOnlyVolume` is `true`.

Reply:
Oh, I missed the correlation between this block and the attributes supported by Google.
Could you guide me on where would be the right place to register an attribute that used only internally?
Essentially this is a config flag used for modifying the behavior of `setVolume` only.

Should this line just be removed?

Reply:
The map returned by this method is the attributes sent to Google in the SYNC request.  The trait map returned by `traitFromSettings_Volume` is the internal representation.  I think this line can just be removed, but I'm not sure the `volumeRelative` command will ever be sent unless this method sets the `commandOnlyVolume` attribute to `true`.  That's what [the docs](https://developers.google.com/assistant/smarthome/traits/volume#action.devices.commands.volumerelative) say, at least.

Reply:
Tested to confirm, and indeed it's unneeded.
Removed.

As for `commandOnlyVolume`, everything is working very nicely also without it.
I believe adding support for it may add the visual volume changer on the app, but that's a separate PR I think, because it requires further adjustments in additional areas.